### ISSUE TITLE
Multi-GPU?

### ISSUE THREAD

**Issue by Sino0904** [AmazingAng/pytorch-tvnet#4]
Hello,

Thanks for the code! Can you tell if we can run this code with multi-gpu setup?

Thanks!

**Comment by AmazingAng** [comment]
I haven't tried, but I think it can be implemented in multi-gpu setup if the data loader is carefully written.

**Comment by Sino0904** [comment]
Thanks! I tried the simple torch.nn.DataParallel but due to predefined matrices in the tvnet.py, the code was not working.
Can you specifically point out what changes needs to be made in the dataloader?

**Comment by AmazingAng** [comment]
> Thanks! I tried the simple torch.nn.DataParallel but due to predefined matrices in the tvnet.py, the code was not working.
> Can you specifically point out what changes needs to be made in the dataloader?

Yes, there are predefined matrices in the network. From my understanding, to use multi-gpu, you need to allocate one copy of the network on each gpu, and then define a dataloader that allocate input to each gpu.
